src/custom-elements/src/components/spinner/spinner.scss

```diff
@@ -0,0 +1 @@
+@import 'components/spinner/spinner';
```

Comment:
Rules should be inside the next wrapper to prevent chi normal rules to have priority over custom-elements rules. In this component might not have sense, but adding the ../../global/styles/common file is recommended for the rest. 
```
:host(chi-spinner) {
  @import '../../global/styles/common';
  @import 'components/spinner/spinner;
}